aria/cli/commands.py

```diff
@@ -164,10 +162,11 @@ def _workspace_setup(self):
         return local_storage()
 
     def _parse_blueprint(self, blueprint_path, inputs=None):
-        plan = parse_from_path(blueprint_path)
-        self.logger.info('blueprint parsed successfully')
-        deployment_plan = prepare_deployment_plan(plan=plan.copy(), inputs=inputs)
-        return plan, deployment_plan
+        #plan = parse_from_path(blueprint_path)
```

Comment:
can we implement this functionality using the new parser? if not, let's remove the commented lines